packages/lesswrong/components/common/Header.tsx

```diff
@@ -450,9 +512,24 @@ const Header = ({
                   </Link>
                 </div>
               </Typography>
+              {showEmojisHeader && <div style={{flexGrow: 2}} />}
```

Comment:
This padding is needed to let us click through the section between the site title and the buttons on the right hand side